internal/api/v1/invoice.go

```diff
@@ -341,3 +341,20 @@ func (h *InvoiceHandler) AttemptPayment(c *gin.Context) {
 
 	c.JSON(http.StatusOK, gin.H{"message": "payment processed successfully"})
 }
+
+func (h *InvoiceHandler) GetInvoicePDF(c *gin.Context) {
```

Comment:
Need to add comments here like this to make sure it comes in swagger specs
```
// AttemptPayment godoc
// @Summary Attempt payment for an invoice
// @Description Attempt to pay an invoice using customer's available wallets
// @Tags Invoices
// @Accept json
// @Produce json
// @Param id path string true "Invoice ID"
// @Success 200 {object} gin.H
// @Failure 400 {object} ierr.ErrorResponse
// @Failure 404 {object} ierr.ErrorResponse
// @Failure 500 {object} ierr.ErrorResponse
// @Router /invoices/{id}/payment/attempt [post]
```

Reply:
yes missed this, adding now